service/pom.xml

```diff
@@ -24,6 +24,11 @@
       <artifactId>dropwizard-core</artifactId>
       <version>${dropwizard.version}</version>
     </dependency>
+    <dependency>
+      <groupId>alexa-skills-kit</groupId>
+      <artifactId>alexa-skills-kit</artifactId>
+      <version>1.1</version>
+    </dependency>
```

Comment:
This is redundant. The `voice-interface` module already includes the ASK, and the service module depends on voice-interface.